app/assets/stylesheets/comments.scss

```diff
@@ -22,15 +24,18 @@
     .media { margin: 5px; }
   }
 
+.comments > .comment {
+  padding: 10px 0px 10px 0px;
```

Comment:
- Properties should be ordered border-bottom, margin, padding
- Shorthands of length `4` are not allowed. Value was `10px 0px 10px 0px`
- Shorthand form for property `padding` should be written more concisely as `10px 0px` instead of `10px 0px 10px 0px`
- `0px` should be written without units as `0`